http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala

```diff
@@ -210,10 +217,16 @@ class RuntimeRoutes(runtimeService: RuntimeService[IO], userInfoDirectives: User
   private[api] def deleteRuntimeHandler(userInfo: UserInfo,
                                         googleProject: GoogleProject,
                                         runtimeName: RuntimeName,
+                                        params: Map[String, String],
                                         span: Span): IO[ToResponseMarshallable] = {
     val res = for {
       implicit0(ctx: ApplicativeAsk[IO, AppContext]) <- AppContext.lift[IO](Some(span))
-      _ <- runtimeService.deleteRuntime(userInfo, googleProject, runtimeName)
+      deleteDisk = params
+        .get("deleteDisk")
+        .map(s => if (s == "true") true else false)
+        .getOrElse(true) //if `deleteDisk` is explicitly set to true, then we delete disk; otherwise, we don't
```

Comment:
why do we say `.getOrElse(true)`, if `deleteDisk` is not in `params` then wouldn't we want to set `deleteDisk` to false so we don't delete the disk?

Reply:
ah nice catch 👍